docs/build/client_example.html

```diff
@@ -265,6 +265,30 @@ <h2>Initializing a model<a class="headerlink" href="#initializing-a-model" title
 </li>
 <li><p>Once all relevant fields are filled out, click “POST” and your model aggregation is created.</p></li>
 </ul>
+<section id="dataprofiler-example">
+<h3>Dataprofiler Example<a class="headerlink" href="#dataprofiler-example" title="Permalink to this heading">#</a></h3>
+<p>A demonstration of the federated model aggregation service for the dataprofiler.</p>
+<section id="running-the-example">
+<h4>Running the Example<a class="headerlink" href="#running-the-example" title="Permalink to this heading">#</a></h4>
+<ol class="arabic simple">
+<li><p>Make sure the FMA service is up and running via this <span class="xref myst">server setup readme</span></p></li>
+<li><p>Upload custom model weight extraction function to <span class="xref myst">model_utils.py</span></p></li>
+<li><p>Run example, specifying the desired number of clients (n=3), user (abc123), the base url the service is at (http://localhost:8000),
+the number of epochs to run the experiment (1), and the train entities (None) to be included in the experiment.</p></li>
+</ol>
+<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">bash</span> <span class="n">run_example</span><span class="o">.</span><span class="n">sh</span> <span class="mi">3</span> <span class="n">abc123</span> <span class="n">http</span><span class="p">:</span><span class="o">//</span><span class="n">localhost</span><span class="p">:</span><span class="mi">8000</span> <span class="mi">1</span> <span class="kc">None</span>
+</pre></div>
+</div>
+<section id="details-on-run-example-sh">
+<h5>Details on run_example.sh:<a class="headerlink" href="#details-on-run-example-sh" title="Permalink to this heading">#</a></h5>
+<ol class="arabic simple">
+<li><p>Generates initial model weights and schema via <span class="xref myst">create_initial_model_json_files.py</span></p></li>
+<li><p>POSTS model registration</p></li>
+<li><p>Executes <span class="xref myst">run_client.sh</span> (n) times</p></li>
+</ol>
+</section>
+</section>
+</section>
```

Comment:
changing from a PR that @ksneab7 made to main a month ago -- validated this is expected